src/Module/Contact.php

```diff
@@ -279,8 +279,6 @@ public static function content(array $parameters = [], $update = 0)
 			if ($contact['network'] == Protocol::PHANTOM) {
 				$contact = false;
 			}
-
-			$contact = ModelContact::checkAvatarCacheByArray($contact);
```

Comment:
Is the `ModelContact` used anywhere else? If yes, can you please replace it with `Model\Contact` and remove the alias declaration?

Reply:
That had been automatically generated by my IDE.